### DISCUSSION TITLE
提示Cannot read properties of undefined (reading 'text')

### DISCUSSION THREAD

**Discussion by bigmalin** [ZuodaoTech/everyone-can-use-english#690] [Q&A]
用openai的api，没成功，提示Cannot read properties of undefined (reading 'text')

**Comment by an-lee** [comment]
用的是官方 key？还是买的中转服务？

**Reply by an-lee** [reply]
如果你填了`接口地址`，可能是需要在末尾加 `/v1`

**Reply by bigmalin** [reply]
末尾加 /v1也没用。。。

**Reply by zlcolin** [reply]
我也遇到问题，加/v1 可以工作
https://iamcolin.xlog.app/eng
